client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -89,10 +114,20 @@ export default class ProductFormDetailsCard extends Component {
 						onRemove={ this.onImageRemove }
 					/>
 					<div className="products__product-form-details-basic">
-						<FormFieldSet>
+						<FormFieldSet className="products__product-form-details-basic-name">
 							<FormLabel htmlFor="name">{ __( 'Product name' ) }</FormLabel>
 							<FormTextInput name="name" value={ product.name || '' } onChange={ this.setName } />
 						</FormFieldSet>
+						<FormFieldSet className="products__product-form-details-basic-sku">
+							<FormLabel htmlFor="sku">{ __( 'SKU:' ) }</FormLabel>
```

Comment:
The sku input needs to have the id `sku` if you're using `htmlFor` here — the `for` attribute matches to the `id` of the input, not the name (the same is true of the other labels on this page). Right now this isn't doing anything — and you can check this by clicking on the label text, if it's correctly linked, i'll put focus into the input.